src/Sulu/Bundle/WebsiteBundle/Tests/Functional/CachingTest.php

```diff
@@ -27,6 +27,7 @@ public static function getSuluContext(): string
     public function testFirstRequestIsACacheMiss()
     {
         $this->purgeDatabase();
+        $this->initPhpcr();
```

Comment:
messenger makes this test fails. interesting this does fix it but not sure why it works before